Podcast/SettingsViewController.swift

```diff
@@ -51,12 +59,46 @@ class SettingsViewController: ViewController, UITableViewDataSource, UITableView
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: reusableCellID, for: indexPath) as? SettingsTableViewCell ?? SettingsTableViewCell()
-        cell.setTitle(sectionsAndItems[indexPath.section][indexPath.row])
+        let setting = sectionsAndItems[indexPath.section][indexPath.row]
+        cell.setTitle(setting.rawValue)
+        switch setting {
+        case .ChangeUsername:
+            cell.accessoryType = .disclosureIndicator
+            break
+        case .PushNotifications:
+            cell.accessoryType = .none
+            break
+        }
         return cell
     }
     
     func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
         return sectionSpacing
     }
+    
+    func tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool {
+        let setting = sectionsAndItems[indexPath.section][indexPath.row]
+        switch setting {
+        case .ChangeUsername:
+            return true
+        case .PushNotifications:
+            return false
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let setting = sectionsAndItems[indexPath.section][indexPath.row]
+        switch setting {
+        case .ChangeUsername:
+            // TODO: Bring up change username
```

Comment:
hmm .. does this interaction look weird?

Reply:
Bringing up the view controller? Yes the view controller looks wicked out of place.

Reply:
LOL. ok let's discuss with Eileen on a better interaction design. We can leave this in for now but should change it soon